modules/virt-about-live-migration.adoc

```diff
@@ -11,10 +11,6 @@ Live migration is the process of moving a running virtual machine instance (VMI)
 
 You can use live migration if the following conditions are met:
 
-* Virtual machines must have a persistent volume claim (PVC) with a shared `ReadWriteMany` (RWX) access mode to be live migrated.
-
-* The pod network binding must not be of the bridge interface type `()`.
-
-* Live migration is supported for virtual machines that are attached to an SR-IOV network interface.
-
-* If the virtual machine uses a host model CPU, live migration is supported only between nodes that support the virtual machine's host model CPU.
+* Shared storage with `ReadWriteMany` (RWX) access mode
```

Comment:
Add a period after the sentence fragment per the IBM [style guidelines](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists).

Also applies to the "Preparing your cluster for OpenShift Virtualization" assembly.